modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,615 @@
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    signers.kms.auth.token-path: /etc/kms-secrets/KMS_AUTH_TOKEN
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - name: tekton-chains-controller
+                  volumeMounts:
+                  - mountPath: /etc/kms-secrets
+                    name: kms-secrets
+                volumes:
+                -  name: kms-secrets
+                   secret:
+                    secretName: kms-secrets
+# ...
+----
+
+* With this update, you can configure a list of namespaces as an argument to the {tekton-chains} controller. If you provide this list, {tekton-chains} watches pipeline runs and task runs only in the specified namespaces. If you do not provide this list, {tekton-chains} watches pipeline runs and task runs in all namespaces.
++
+.Example configuration for watching only the `dev` and `test` namespaces
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  chain:
+    disabled: false
+    options:
+      deployments:
+        tekton-chains-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                - args:
+                  - --namespace=dev, test 
+                  name: tekton-chains-controller
+# ...
+----
+
+[id="tekton-results-new-features-1-16_{context}"]
+=== {tekton-results}
+
+* Before this update, {tekton-results} used the `v1beta1` API format to store `TaskRun` and `PipelineRun` object records. With this update, {tekton-results} uses the `v1` API format to store `TaskRun` and `PipelineRun` object records.
+
+* With this update, {tekton-results} can automatically convert existing records to the `v1` API format. To enable such conversion, set parameters in the `TektonResult` CR as shown in the following example:
++
+.Example of configuring {tekton-results} to convert existing records to the `v1` API format
+[source,yaml]
+----
+  apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonResult
+metadata:
+  name: result
+spec:
+  options:
+    deployments:
+      tekton-results-api:
+        spec:
+          template:
+            spec:
+              containers:
+                - name: api
+                  env:
+                    - name: CONVERTER_ENABLE
+                      value: "true"
+                    - name: CONVERTER_DB_LIMIT
+                      value: "256" #<1>
+# ...
+----
+<1> In the `CONVERTER_DB_LIMIT` variable, set the number of records to convert at the same time in a single transaction.
+
+* With this update, {tekton-results} now supports fetching forwarded logs from third party logging APIs. You can enable the logging API through the `TektonResult` CR by setting the `logs_api` to `true` and `logs_type` to `Loki`.
```

Comment:
Either "the `logs_api` field" (or similar word) or just drop "the", which seems more consistent with the rest of these relnotes:
```suggestion
* With this update, {tekton-results} now supports fetching forwarded logs from third party logging APIs. You can enable the logging API through the `TektonResult` CR by setting `logs_api` to `true` and `logs_type` to `Loki`.
```